docs/GettingStartedDocs/IntelSDKPortingGuide.md

```diff
@@ -0,0 +1,214 @@
+# Intel SGX SDK to OpenEnclave SDK Porting Guide
+
+Intel SGX SDK and OpenEnclave SDK share many design principles but differ in implementations. It requires source code changes to build enclaves with OpenEnclave SDK that were developed initially with Intel SGX SDK.
+
+## Create Project Build Files
+
+OpenEnclave SDK uses `cmake` as the build system, so the first step of building an enclave is to create a `CMakeLists.txt` file for it. For those not familiar with `cmake`, a tutorial is available at https://cmake.org/cmake/help/latest/guide/tutorial/index.html.
```

Comment:
This statement is not true for Windows.  Don't confuse the build system for the SDK itself, with the build system for apps.
For enclaves built using the SDK, there are multiple build systems supported.  For example, for Visual Studio as a build system, see https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/VisualStudioWindows.md